### PR TITLE
ONI-30: set contribution amount fixed on config

### PR DESCRIPTION
**Pull request by olewandowski1**
[ONI-30](https://openimis.atlassian.net/browse/ONI-30)

Changes:
- Added possibility of disabling multiple contributions. If 'isMultiplePaymentsAllowed' is false, the user must make a single payment, fixed at the policy value. If true, the user can pay in installments.

[ONI-30]: https://openimis.atlassian.net/browse/ONI-30?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by wzglinieckisoldevelo** [comment]
LGTM

**Comment by wzglinieckisoldevelo** [comment]
Based on:
https://github.com/openimis/openimis-fe-contribution_js/pull/29
Which is now closed.

**Review by sniedzielski — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -39,4 +39,4 @@ None
 * `state.insuree`, loading insuree policies (,eligibility,...)
 
 ## Configurations Options
-None
\ No newline at end of file
+- `isMultiplePaymentsAllowed`: Controls if the user is permitted to make installment payments. If __false__, user must make a single payment, fixed at the policy value. If __true__, user can pay off in installments. Default __true__.
```
Comment (wzglinieckisoldevelo) [inline README.md:42]: Do not merge it, this solution is not correct, but can be useful as a future reference.

Comment (dragos-dobre) [reply README.md:42]: @wzglinieckisoldevelo if you don't want this PR to be merged, please make it a draft

**Pull request closed without merging**